### PR TITLE
docs(rxTags): Page objects to jsdoc3

### PR DESCRIPTION
**Pull request by Droogans**
### LGTMs
- [x] Dev LGTM
- [x] QE LGTM

### CODE REVIEW

File: src/components/rxTags/scripts/rxTags.page.js [hunk 1]
```diff
@@ -88,6 +196,19 @@ var rxTags = {
...
+     * @instance
+     * @function
+     * @description Adds a new tag with text `text` to the group of tags.
+     * Returns the newly created tag, should you need to interact with it.
+     * @param {String} text - The desired text of the tag to be created.
+     * @returns {rxTags.tag}
+     * @example
+     * it('should show an warning notification when adding the "Enterprise" tag', function () {
+     *     encore.rxTags.initialize().addTag('Enterprise');
+     *     expect(encore.rxNotify.all.exists('Warning: "Enterprise"', 'warning')).to.eventually.be.true;
```
Comment (CITguy) [inline src/components/rxTags/scripts/rxTags.page.js]: Please extract actual value out of `expect(...)` to a variable. The example code in documentation is scrolling out of view.

```javascript
// ...
var actual = encore.rxNotify.all.exists('Warning: "Enterprice"', 'warning');
expect(actual).to.eventually.be.true;
```

File: src/components/rxTags/scripts/rxTags.page.js [hunk 2]
```diff
@@ -114,6 +238,19 @@ var rxTags = {
...
+     * @function
+     * @instance
+     * @returns {rxTags.tag}
+     * @description Return an {@link rxTags.tag} object that matches the `text` passed in.
+     * This function uses a partial text match function to find the tag, so make sure there
+     * are no duplicates. If there are, the first matching tag will be returned.
+     * @param {String} text - The text of the tag to look for and return as a {@link rxTags.tag} object.
+     * @example
+     * it('should have the correct category for the "Strawberry" tag', function () {
+     *     expect(encore.rxTags.initialize().byText('Strawberry').category).to.eventually.equal('fruit');
```
Comment (CITguy) [inline src/components/rxTags/scripts/rxTags.page.js]: Please perform similar update to example as mentioned above. The example source is scrolling out of view in the documentation.

**Comment by CITguy** [comment]
@Droogans, we'll need to trigger another Travis build, somehow. It seems the Travis job was lost.

**Comment by jj4th** [comment]
QE-LGTM

**Pull request merged**